cpp/velox/shuffle/VeloxSplitter.h

```diff
@@ -99,40 +101,12 @@ class VeloxSplitter {
   static arrow::Result<std::shared_ptr<VeloxSplitter>>
   Make(const std::string& name, uint32_t num_partitions, SplitOptions options = SplitOptions::Defaults());
 
-  virtual arrow::Status Split(const facebook::velox::RowVector& rv);
+  virtual arrow::Status Split(std::shared_ptr<ColumnarBatch> cb);
```

Comment:
you may use `ColumnarBatch* cb` or `ColumnarBatch&` instead
[smart](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-smart)